includes/Admin/Editor.php

```diff
@@ -370,6 +370,7 @@ public function get_editor_settings(): array {
 
 		$settings = [
 			'autoSaveInterval'        => \defined( 'AUTOSAVE_INTERVAL' ) ? AUTOSAVE_INTERVAL : null,
+			'localAutoSaveInterval'   => \defined( 'LOCAL_AUTOSAVE_INTERVAL' ) ? LOCAL_AUTOSAVE_INTERVAL : null,
```

Comment:
@swissspidy Any thoughts on how it would be the best to define the local autosave interval? Added it here for now but maybe it should come from somewhere else, e.g. defined when initing the app. Thoughts?

Reply:
I'd just hardcode it to a number. WP does that as well.

It can be changed via the `web_stories_editor_settings` filter below.

```suggestion
			'localAutoSaveInterval'   => 15,
```